### PR TITLE
Import fenix quarantine

### PR DESCRIPTION
**Pull request by mozilla-pontoon**

### CODE REVIEW

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -356,13 +362,17 @@
     <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is on for a site, this is shown as part of the cookie banner panel in the toolbar. The first and second parameter are the application name -->
     <string name="reduce_cookie_banner_details_panel_description_on_for_site" moz:RemovedIn="110" tools:ignore="UnusedResources">%1$s can try to automatically reject cookie requests. If a reject option isn’t available, %2$s may accept all cookies to dismiss the banner.</string>
     <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is on for a site, this is shown as part of the cookie banner panel in the toolbar. The first and second parameter are the application name -->
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml:364]: seems like the comment here should be updated as there is only one parameter in the string it pertains to

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -356,13 +362,17 @@
...
-    <string name="reduce_cookie_banner_details_panel_description_on_for_site_1">%1$s can try to automatically reject cookie requests.</string>
+    <string name="reduce_cookie_banner_details_panel_description_on_for_site_1" moz:RemovedIn="111" tools:ignore="UnusedResources">%1$s can try to automatically reject cookie requests.</string>
+    <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is on for a site, this is shown as part of the cookie banner panel in the toolbar. The first and second parameter are the application name -->
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml:366]: Comment update: only one parameter in the string below as well

**Review by Delphine — CHANGES_REQUESTED** [review]
@Amejia481: Just a few comments to update to avoid any possible confusion on localizer side.
Thank you!

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Amejia481) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:364]: Thanks for catching this 😅

**Review by Delphine — APPROVED** [review]
LGTM

**Pull request merged**